test_api.py

```diff
@@ -0,0 +1,128 @@
+import requests
+import sqlite3
+
+from datetime import datetime
+# from pprint import pprint as pp
+
+
+DB = 'test.db'
+ID_G = iter(range(1, 10_000))
+
+init_data = [
+    (next(ID_G), 'Prepare a dinner for a week', 'home',
```

Comment:
Good idea for using generator

Reply:
I "stole" this idea from one of the homework of Aleksandr Koryagin. I'd like this trick, and i thought that here it is a suitable case for use it. :)